Source/Scene/Cesium3DTile.js

```diff
@@ -206,14 +231,17 @@ define([
         },
 
         /**
-         * Get the tight oriented bounding box
+         * Get the bounding volume of the tile
          *
-         * @type {Promise}
+         * @type {Object}
          * @readonly
          */
-        orientedBoundingBox : {
+        boundingVolume : {
```

Comment:
For the original name question, when the content bounding volume is not defined, it defaults to the tile's bounding volume, so this is the content's bounding volume even if it returns the tile's bounding volume.